src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -25,6 +25,16 @@ class FacilityDetail extends Component {
     focusElement('.va-nav-breadcrumbs');
   }
 
+  componentDidUpdate(prevProps) {
+    const facilityJustLoaded = !prevProps.facility && this.props.facility;
```

Comment:
Is this enough to change the title to the right facility name? I noticed in my local testing that choosing one result and going back to the results keeps the title as the facility name. And then going to another facility in the results doesn't change the title to the new facility.

Reply:
Ah man, great point! Thanks for finding that. I’ll patch that in the morning.